pkgs/development/libraries/kmsxx/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, cmake, pkgconfig, libdrm, pythonBindings ? null }:
```

Comment:
I suggest to just pass in `python` and build the bindings regardless. We could (perhaps later) put them in a separate output.